nvme_gw_cli.py

```diff
@@ -0,0 +1,249 @@
+# Copyright (c) 2021 International Business Machines
+# All rights reserved.
+
+# SPDX-License-Identifier: LGPL-3.0-or-later
+
+# Authors: anita.shekar@ibm.com, sandy.kaur@ibm.com
+
+import argparse
+import grpc
+import nvme_gw_pb2_grpc as pb2_grpc
+import nvme_gw_pb2 as pb2
+import nvme_gw_config
+
+
+def argument(*name_or_flags, **kwargs):
+    #
+    # Helper function to format arguments for argparse command decorator.
+    #
```

Comment:
Ah, I only meant the license comment since that is how you did it in other files.  Documentation strings should use `"""` syntax per https://www.python.org/dev/peps/pep-0008/#documentation-strings.  Some reverting to do here, sorry for the confusion!